package.json

```diff
@@ -19,9 +19,12 @@
   "dependencies": {
     "@openzeppelin/contracts": "^4.8.3",
     "@types/express": "~4.17.13",
+    "@types/multer": "^1.4.7",
     "axios": "^1.0.0",
     "express": "~4.18.1",
+    "fs": "^0.0.1-security",
```

Comment:
You don't need to install this, fs module is part of node https://nodejs.org/api/fs.html